app/Http/Controllers/PhotoController.php

```diff
@@ -27,22 +26,29 @@ public function store(Request $request)
         return redirect()->back()->with('success', 'Fotoğrafınız Kaydedildi');
     }
 
-    public function parcelStore(Request $request)
+    public function parcelStore(Request $request, EstateProject $project)
     {
```

Comment:
They wanted to add another structure instead of parcel. So this is not currently in use.
We are using the numbering object. Which is created manually.